app/models/task.py

```diff
@@ -1,5 +1,46 @@
 from app import db
+from datetime import datetime
 
 
 class Task(db.Model):
     task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, nullable=True)
+    is_complete = False
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'))
+    goal = db.relationship("Goal", back_populates="tasks")
+
+
+    def to_json(self):
+        task_dict = { 
+                "id": self.task_id,
+                "title": self.title,
+                "description": self.description,
+                "is_complete" : self.is_complete
+                }
+        
+        if self.goal_id:
+            task_dict["goal_id"] = self.goal_id
+
+        return task_dict
+
+    def written_out(cls):
+        return "task"
```

Comment:
I think this would make for a good `@staticmethod` over a `@classmethod`

```suggestion
    @staticmethod
    def written_out():
        return "task"
```

Reply:
also is this used anywhere? I couldn't find any files where this method was called? If that's the case, let's get rid of it